CONTRIBUTING.md

```diff
@@ -1,3 +1,218 @@
+    are public and that a record of the contribution (including all
+    personal information I submit with it, including my sign-off) is
+    maintained indefinitely and may be redistributed consistent with
+    this project or the open source license(s) involved.
+```
+
+Then you just add a line to every git commit message:
+
+    Signed-off-by: Joe Smith <joe.smith@example.com>
+
+Use your real name (sorry, no pseudonyms or anonymous contributions.)
+
+If you set your `user.name` and `user.email` git configs, you can sign your commit automatically
+with `git commit -s`.
+
+Note: If your git config information is set properly then viewing the `git log` information for your
+ commit will look something like this:
+
+```
+Author: Joe Smith <joe.smith@example.com>
+Date:   Thu Feb 2 11:41:15 2018 -0800
+
+    Update README
+
+    Signed-off-by: Joe Smith <joe.smith@example.com>
+```
+
+Notice the `Author` and `Signed-off-by` lines match. If they don't your PR will be rejected by the
+automated DCO check.
+
+## Support Channels
+
+Whether you are a user or contributor, official support channels include:
+
+- [Issues](https://github.com/helm/helm-mapkubeapis/issues)
+- Slack:
+  - User: [#helm-users](https://kubernetes.slack.com/messages/C0NH30761/details/)
+  - Contributor: [#helm-dev](https://kubernetes.slack.com/messages/C51E88VDG/)
+
+Before opening a new issue or submitting a new pull request, it's helpful to search the project -
+it's likely that another user has already reported the issue you're facing, or it's a known issue
+that we're already aware of. It is also worth asking on the Slack channels.
+
+## Semantic Versioning
+
+Helm maintains a strong commitment to backward compatibility. All of our changes to protocols and
+formats are backward compatible from one major release to the next. No features, flags, or commands
+are removed or substantially modified (unless we need to fix a security issue).
+
+We also remain committed to not changing publicly accessible Go library definitions inside of the `pkg/` directory of our source code in a non-backwards-compatible way.
+
+For more details on Helm’s minor and patch release backwards-compatibility rules, please read [HIP-0004](https://github.com/helm/community/blob/main/hips/hip-0004.md)
+
+For a quick summary of our backward compatibility guidelines for releases between 3.0 and 4.0:
+
+- Command line commands, flags, and arguments MUST be backward compatible
+- File formats (such as Chart.yaml) MUST be backward compatible
+- Any chart that worked on a previous version of Helm 3 MUST work on a new version of Helm 3
+  (barring the cases where (a) Kubernetes itself changed, and (b) the chart worked because it
+  exploited a bug)
+- Chart repository functionality MUST be backward compatible
+- Go libraries inside of `pkg/` MUST remain backward compatible, though code inside of `cmd/` and
+  `internal/` may be changed from release to release without notice.
```

Comment:
This is really for Helm itself (and refers to 3.0 and 4.0 while mapkubeapis isn't even 1.0). Can you delete it?

Reply:
done and updated.